osquery/CMakeLists.txt

```diff
@@ -304,6 +304,10 @@ if(NOT DEFINED ENV{SKIP_TESTS})
   add_executable(osquery_tests main/tests.cpp ${OSQUERY_TESTS})
   ADD_DEFAULT_LINKS(osquery_tests FALSE)
   target_link_libraries(osquery_tests gtest gmock libosquery_testing)
+  if(WINDOWS)
+    # Linked for the WMI libs
+    target_link_libraries(osquery_tests Release/osquery_additional)
```

Comment:
Done. If this poses a threat I can also try and get the refactor cranked out with this PR? https://github.com/facebook/osquery/issues/2524